templates/taxi/driver_list.html

```diff
@@ -1,7 +1,11 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Driver List
+    <h1>
+      Driver List
+      <a href="{% url 'taxi:driver-create' %}" class="btn btn-primary link-to-page">
+        Create
+      </a>
```

Comment:
Same here: better to put <a> outside header.

Reply:
Thanks, changed.